src/main/kotlin/world/player/item/banking/regularBank/openBank.kts

```diff
@@ -2,11 +2,57 @@ package world.player.item.banking.regularBank
 
 import api.predef.*
 import io.luna.game.event.impl.ServerStateChangedEvent.ServerLaunchEvent
+import io.luna.game.model.mob.*
+
+val bankerIds = listOf(166, 494, 495, 496, 497, 498, 499, 953, 1036, 1360, 1702, 2163, 2164, 2354, 2355, 2568, 2569, 2570, 3046, 3198, 3199)
```

Comment:
Add a TODO to dynamically generate this in the future (ie. all npcs with name "banker")